src/drawers/areaDrawer.ts

```diff
@@ -16,7 +16,7 @@ export module Drawers {
     }
 
     public selectionForIndex(index: number) {
-      return this.renderArea().select(this.selector());
```

Comment:
I essentially mean `this.renderArea().select(this.selection()[0][0])`

Whenever I see `d3.select`, I personally get scared that it'll look throughout the entire page for the selector, which can lead to bugs sometimes